CHANGES.rst

```diff
@@ -10,6 +10,24 @@
 
 .. towncrier release notes start
 
+3.10.6rc1 (2024-09-22)
+======================
+
+Removals and backward incompatible breaking changes
+---------------------------------------------------
+
+- Increased minimum yarl version to 1.12.0 -- by :user:`bdraco`.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`9267`.
```

Comment:
This changelog is linked to #8898 and will appear that way in the stable release